### PR TITLE
CVE-2016-0763

### PR DESCRIPTION
**Pull request by TLezynski**

### CODE REVIEW

File: cves/CVE-2016-0763.yml [hunk 1]
```diff
@@ -4,27 +4,27 @@ nickname_instructions: |
...
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: global shouldn't mean globl
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: Bad Coding Practices
```
Comment (mesternefeld) [inline cves/CVE-2016-0763.yml]: Please put the CWE number if possible

File: cves/CVE-2016-0763.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ upvotes:
 unit_tested:
   question: |
     Were automated unit tests involved in this vulnerability?
-    Was the original code unit tested, or not unit tested? Did the fix involve
+        Was the original code unit tested, or not unit tested? Did the fix involve
```
Comment (mesternefeld) [inline cves/CVE-2016-0763.yml]: Please fix it to the original

File: cves/CVE-2016-0763.yml [hunk 3]
```diff
@@ -120,7 +120,7 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
+  answer: I was unable to find this information.
```
Comment (akn4743) [inline cves/CVE-2016-0763.yml]: Even though you couldn't find the fix, add places where you looked. Any reputable blog or forum would be good to write down.

File: cves/CVE-2016-0763.yml [hunk 4]
```diff
@@ -4,27 +4,27 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: global shouldn't mean globl
```
Comment (akn4743) [inline cves/CVE-2016-0763.yml]: global*

**Review by akn4743 — CHANGES_REQUESTED** [review]

File: cves/CVE-2016-0763.yml (hunk 4 above)
Comment (TLezynski) [reply cves/CVE-2016-0763.yml]: Thanks for catching that!

File: cves/CVE-2016-0763.yml (hunk 1 above)
Comment (TLezynski) [reply cves/CVE-2016-0763.yml]: Added

File: cves/CVE-2016-0763.yml (hunk 2 above)
Comment (TLezynski) [reply cves/CVE-2016-0763.yml]: Fixed

File: cves/CVE-2016-0763.yml (hunk 3 above)
Comment (TLezynski) [reply cves/CVE-2016-0763.yml]: Thanks, I updated this

File: cves/CVE-2016-0763.yml [hunk 5]
```diff
@@ -132,8 +132,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: factory
```
Comment (PaceyLong) [inline cves/CVE-2016-0763.yml:135]: Please write a full sentence.

**Pull request merged**